features/AuthController.feature

```diff
@@ -56,6 +56,20 @@ Feature: Auth Controller
       | _id        | _source.userId | _source.ttl | _source.expiresAt | _source.description | _source.fingerprint |
       | "_STRING_" | "test-admin"   | -1          | -1                | "Sigfox API key"    | "_STRING_"          |
 
+  # auth:login
+
+  @security
+  Scenario: Login with cookieOnly
+    When I successfully execute the action "auth":"login" with args:
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
```

Comment:
Nitpicking: you can use lodash syntax to construct objects
```suggestion
      | body.username       | "test-admin" |
      | body.password        | "password"  |
```

Reply:
This seems not to be working